pkgs/pkgs-lib/formats.nix

```diff
@@ -274,6 +274,85 @@ rec {
 
   };
 
+  # Format for defining configuration of some PHP services, that use "config.php" approach.
+  php = {finalVariable}: let
+    phpLib = {
+      phpFile = x: "<?php\n" + x;
```

Comment:
Shouldn't there be a closing tag?

Reply:
Nope, [see this](https://www.php.net/manual/en/language.basic-syntax.phptags.php#:~:text=If%20a%20file,in%20the%20script.).